python/tests/sklearn/preprocessing/test_one_hot_encoder.py

```diff
@@ -0,0 +1,145 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+import os
+import shutil
+import unittest
+import uuid
+
+import numpy as np
+
+from mleap.sklearn.preprocessing.data import LabelEncoder
+from mleap.sklearn.preprocessing.data import OneHotEncoder
+
+
+class TestOneHotEncoder(unittest.TestCase):
+    def setUp(self):
+        labels = ['a', 'b', 'c', 'a', 'b', 'b']
+        self.le = LabelEncoder(input_features=['label'], output_features='label_le_encoded')
+        self.oh_data = self.le.fit_transform(labels).reshape(-1, 1)
+
+        self.tmp_dir = "/tmp/mleap.python.tests/{}".format(uuid.uuid1())
+        if os.path.exists(self.tmp_dir):
+            shutil.rmtree(self.tmp_dir)
+        os.makedirs(self.tmp_dir)
+
+    def tearDown(self):
+        shutil.rmtree(self.tmp_dir)
+
+    def test_one_hot_encoder_serialization_fails_on_multiple_feature_columns(self):
+        self.oh_data = np.hstack((self.oh_data, self.oh_data))  # make two feature columns
+
+        ohe = OneHotEncoder(handle_unknown='error')
+        ohe.mlinit(prior_tf=self.le, output_features='{}_one_hot_encoded'.format(self.le.output_features))
+        ohe.fit(self.oh_data)
```

Comment:
I'm a bit surprised that this doesn't throw an error here since handle_unknown="error"

Reply:
handle_unknown only matters if you call `transform` on labels that weren't seen during `fit`. So this doesn't raise here because `fit` is what determines which classes are unknown in the first place